### PR TITLE
Add improvement radar links

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

File: openprescribing/frontend/tests/functional/test_measures.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def test_measures_for_one_practice(self):
         self._verify_link(
             panel_element,
             ".inner li:nth-child(3)",
-            "Compare all Sub-ICB Locations in England on this measure",
+            "Compare all SICBLs in England on this measure",
```
Comment (evansd) [inline openprescribing/frontend/tests/functional/test_measures.py:322]: How did these pass before?

**Review by evansd — APPROVED** [review]
Surgical!

Although it looks like the tests are complaining about trying to stick the string `"false"` in a boolean field. I can't quite see why this should be happening though.

File: openprescribing/frontend/tests/functional/test_measures.py (hunk 1 above)
Comment (inglesp) [reply openprescribing/frontend/tests/functional/test_measures.py:322]: They didn't!

Comment (inglesp) [reply openprescribing/frontend/tests/functional/test_measures.py:322]: Oh!  These tests are failing in CI.  Bamboozling.

Comment (evansd) [reply openprescribing/frontend/tests/functional/test_measures.py:322]: I wonder if you haven't rebuilt your JS locally?

Comment (inglesp) [reply openprescribing/frontend/tests/functional/test_measures.py:322]: 5am thought: out-of-date JS?

Comment (inglesp) [reply openprescribing/frontend/tests/functional/test_measures.py:322]: jinx

**Pull request merged**